operatorcourier/identify.py

```diff
@@ -1,16 +1,27 @@
-import yaml
+from yaml import safe_load
+from yaml import MarkedYAMLError
+import logging
+
+logger = logging.getLogger(__name__)
+
 
 def get_operator_artifact_type(operatorArtifactString):
-    """get_operator_artifact_type takes a yaml string and determines if it is one of the 
+    """get_operator_artifact_type takes a yaml string and determines if it is one of the
     expected bundle types: ClusterServiceVersion, CustomResourceDefinition, or Package.
 
     :param operatorArtifactString: Yaml string to type check
     """
-    operatorArtifact = yaml.safe_load(operatorArtifactString)
-    if isinstance(operatorArtifact, dict):
-        if "packageName" in operatorArtifact:
-            return "Package"
-        elif "kind" in operatorArtifact:
-            if operatorArtifact["kind"] in ("ClusterServiceVersion", "CustomResourceDefinition"):
-                return operatorArtifact["kind"]
-    raise ValueError('Courier requires valid CSV, CRD, and Package files')
+    try:
+        operatorArtifact = safe_load(operatorArtifactString)
+    except MarkedYAMLError:
+        logger.error("Invalid YAML file")
+        print('Courier requires valid input YAML files')
+        exit(0)
```

Comment:
Rather than `exit(0)` here, can we raise an exception like later in the file?

Reply:
or at least exit non-zero...in addition, with #27 ... this should be a critical error in the json output. @awgreene

Reply:
@scoheb I have discussed with @kevinrizza. Instead of raising an exception, we will just log error messages to the output. And this will be made consistent in the entire program.

Reply:
Please never ever call `exit()` in any python module outside CLI processing. It will cause immediate exit from any app using `operator-courier` as its part

Reply:
I now only exit(1) in the main entry of the CLI, if any error bubbles up to this level.